src/utils.jl

```diff
@@ -0,0 +1,49 @@
+StatsBase.describe(X::NullableVector) = StatsBase.describe(STDOUT, X)
+
+function StatsBase.describe(io::IO, X::NullableVector{<:Real})
```

Comment:
Since this package supports 0.5 (unlike DataArrays), the `T{<:S}` syntax won't work. Not sure whether there's an `@compat` for it, but you could just do
```julia
function StatsBase.describe{T<:Real}(io::IO, X::NullableVector{T})
```